targets/csharp/source/source/PlayFabGameServerHttp.cs

```diff
@@ -0,0 +1,75 @@
+﻿////////////////////////////////////////////////
+// Copyright (C) Microsoft. All rights reserved.
+////////////////////////////////////////////////
+
+// GSDK Only has the following supported frameworks for now
+#if NETSTANDARD2_0 || NETCOREAPP2_1
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+
+namespace PlayFab.GSDK
+{
+    // TODO: This should eventually be merged with PlayFabHttp
```

Comment:
This has to be done before this can be merged with the SDK.  We won't allow separate HTTP stacks.